### PR TITLE
Add README on linux dGPU

### PR DESCRIPTION
**Pull request by Zhangky11**

### CODE REVIEW

File: README_linux_dgpu.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  > When creating the conda environment we used python 3.11, which is different from the default recommended python version 3.9 in [Install IPEX-LLM on Linux with Intel GPU Guide](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/install_linux_gpu.html)
+
+
+#### 2.  Install `ipex-llm` 
+  ```cmd
+  pip install --pre --upgrade ipex-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+  pip install --pre --upgrade torchaudio==2.1.0a0  -f https://developer.intel.com/ipex-whl-stable-xpu
+  ```
+#### 3. Install Langchain-Chatchat Dependencies 
+Switch to the root directory of Langchain-Chatchat you've downloaded (refer to the [download section](#download-langchain-chatchat)), and install the dependencies with the commands below. **Note: In the example commands we assume the root directory is `/home/usr/Langchain-Chatchat-ipex-llm`. Remember to change it to your own path**).
```
Comment (Oscilloscope98) [inline README_linux_dgpu.md]: Let's change usr -> arda as we done that for windows mtl igpu guide.

File: README_linux_dgpu.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+| Model |Category| download link | 
+|:--|:--|:--|
+|`THUDM/chatglm3-6b`|Chinese LLM| [HF](https://huggingface.co/THUDM/chatglm3-6b) or [ModelScope](https://www.modelscope.cn/models/ZhipuAI/chatglm3-6b/summary) |
+|`meta-llama/Llama-2-7b-chat-hf`|English LLM| [HF](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) | 
+|`BAAI/bge-large-zh-v1.5`|Chinese Embedding| [HF](https://huggingface.co/BAAI/bge-large-zh-v1.5) |
+|`BAAI/bge-large-en-v1.5`| English Embedding|[HF](https://huggingface.co/BAAI/bge-large-en-v1.5) |
+
+## Start the Service
+Run the following commands on **Intel Arc™ A-Series Graphics** with the exception of Intel Arc™ A300-Series or Pro A60:
```
Comment (Oscilloscope98) [inline README_linux_dgpu.md]: Put these device related info in a note box below the configuration?

E.g.

>[!Note]
> The above configurations lead to optimal performance for **Intel Arc™ A-Series Graphics** with the exception of Intel Arc™ A300-Series or Pro A60.

```
>[!Note]
> The above configurations lead to optimal performance for **Intel Arc™ A-Series Graphics** with the exception of Intel Arc™ A300-Series or Pro A60.
```

File: README_linux_dgpu.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+#### 2.  Install `ipex-llm` 
+  ```cmd
+  pip install --pre --upgrade ipex-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+  pip install --pre --upgrade torchaudio==2.1.0a0  -f https://developer.intel.com/ipex-whl-stable-xpu
+  ```
+#### 3. Install Langchain-Chatchat Dependencies 
+Switch to the root directory of Langchain-Chatchat you've downloaded (refer to the [download section](#download-langchain-chatchat)), and install the dependencies with the commands below. **Note: In the example commands we assume the root directory is `/home/usr/Langchain-Chatchat-ipex-llm`. Remember to change it to your own path**).
+  ```cmd
```
Comment (Oscilloscope98) [inline README_linux_dgpu.md]: cmd -> bash

File: README_linux_dgpu.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+| Model |Category| download link | 
+|:--|:--|:--|
+|`THUDM/chatglm3-6b`|Chinese LLM| [HF](https://huggingface.co/THUDM/chatglm3-6b) or [ModelScope](https://www.modelscope.cn/models/ZhipuAI/chatglm3-6b/summary) |
+|`meta-llama/Llama-2-7b-chat-hf`|English LLM| [HF](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) | 
+|`BAAI/bge-large-zh-v1.5`|Chinese Embedding| [HF](https://huggingface.co/BAAI/bge-large-zh-v1.5) |
+|`BAAI/bge-large-en-v1.5`| English Embedding|[HF](https://huggingface.co/BAAI/bge-large-en-v1.5) |
+
+## Start the Service
+Run the following commands on **Intel Arc™ A-Series Graphics** with the exception of Intel Arc™ A300-Series or Pro A60:
+```cmd
```
Comment (Oscilloscope98) [inline README_linux_dgpu.md]: cmd -> bash

**Comment by Oscilloscope98** [comment]
@shane-huang 

Here are the draft setup guide for Linux dGPU users :)

**Review by shane-huang — APPROVED** [review]
LGTM.

**Pull request merged**